src/main/scala/com/ocadotechnology/Main.scala

```diff
@@ -0,0 +1,33 @@
+package com.ocadotechnology
+
+import cats.effect.{ExitCode, IO, IOApp}
+import com.comcast.ip4s.{Host, Port, port}
+import org.http4s.ember.server.EmberServerBuilder
+import org.http4s.server.Router
+import sttp.tapir.server.http4s.Http4sServerInterpreter
+
+object Main extends IOApp:
+
+  override def run(args: List[String]): IO[ExitCode] =
+
+    val routes = Http4sServerInterpreter[IO]().toRoutes(Endpoints.all)
+
+    val port = sys.env
+      .get("HTTP_PORT")
+      .flatMap(_.toIntOption)
+      .flatMap(Port.fromInt)
+      .getOrElse(port"8080")
+
+    EmberServerBuilder
+      .default[IO]
+      .withHost(Host.fromString("localhost").get)
```

Comment:
```suggestion
      .withHost(Host.fromString("0.0.0.0").get)
```
With `localhost` we only allow connections from your local machine. If we are to ever host it, we need to listen on all network interfaces, thus this change.